channels/us.m3u

```diff
@@ -6293,9 +6293,9 @@ https://venntv-rakuten-samsung.amagi.tv/playlist.m3u8
 https://venntv-roku.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Entertainment",VENN (1080p)
 https://venntv-samsungus.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="Venn.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Entertainment",VENN
+#EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Entertainment",VENN
 https://venntv-vizio.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="VennTVEngland.us" tvg-name="VENN (England)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Sports",VENN (England)
+#EXTINF:-1 tvg-id="VENNTVEngland.us" tvg-name="VENN (England)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Sports",VENN (England)
```

Comment:
tvg-id shouldn't contain tv and if it's england stream tvg-country is UK

Reply:
Note that it might be linked to the server in which the channel is being streamed, *not* the content.

It might be the same VENN than in the US, just a different server (like the Chilian channels, in which there's both an US and a CL server for the same stream)

Reply:
You guys are good.

Reply:
Otherwise, if the channel does not contain "TV" in it's name, you should take it off. Same for the "England" tag on ID. It's still VENN.

Reply:
Yeah, you can normalize this channel as US and UK streams show the same programming now

Reply:
Indeed, you can do what @Dum4G says.

Just checked, they're the same.